### ISSUE TITLE
About ContaGan 2C loss

### ISSUE THREAD

**Issue by WuDiDaBinGe** [POSTECH-CVLab/PyTorch-StudioGAN#85]
![image](https://user-images.githubusercontent.com/37055010/134762386-55110c03-f9c8-4a2b-8e34-182d14f0b5fb.png)
this is 2c Loss, when i watch the code, i found the 2c loss code  multiply  a extra temperature. 
`denomerator = torch.cat([torch.unsqueeze(inst2proxy_positive, dim=1), instance_zone], dim=1).sum(dim=1) `
` criterion = -torch.log(temperature * (numerator / denomerator)).mean()`

**Comment by WuDiDaBinGe** [comment]
OK.Thanks a lot
